### DISCUSSION TITLE
What does Fixed Image Aspect and Autosize Grid options do?

### DISCUSSION THREAD

**Discussion by healthyfat** [jgraph/drawio#4507] [Q&A]
What does Fixed Image Aspect option in style property do? 
What is the difference between Autosize Grid enabled and disabled? 

Is there a description of all the options in style property?

**Comment by skybitscom** [comment]
Fixed Image Aspect would mean the image has a set image size of x and y pixels. An Autosize Grid would relate to resizing per a percentage number instead of a fixed number.
